### PR TITLE
Connected `Wallet Details` page API

### LINKED ISSUE #34: Wallet details page : getting the wallet details from the backend and displaying them

### PR DESCRIPTION
**Pull request by NikhilSharma03**
## Description

- This PR resolves #34 
- Connected `Wallet Details` page API
- Fixed `Get Started` button issue

### CODE REVIEW

File: pages/manage/[pubkey].tsx [hunk 1]
```diff
@@ -3,69 +3,47 @@ import type { NextPage } from 'next'
...
+      <div className="flex flex-row justify-center items-center">
+        <LoadingSpinner />
+      </div>
+    )
+  }
+
   return (
     <div className="container mx-auto px-6 sm:px-0 gap-10 flex flex-col justify-center items-center my-10">
-      <WalletDetails wallet={wallet} />
+      <WalletDetails wallet={data} />
```
Comment (abdelhakeem) [inline pages/manage/[pubkey].tsx]: The API returns the wallet data in the following format:

```typescript
{
  found: boolean,
  wallet: {
    name: string,
    pubkey: string,
    authority: string,
    memberShipType: string,
    acceptSPL: boolean,
    totalShares: number,
    cluster: string,
    validated: boolean
  },
  members: {
    memberPubkey: string,
    createdAt: Date,
    shareCount: number,
    cluster: string,
    walletPubkey: string
  }[]
}
```

You would check what format the `WalletDetails` component expects the data to be in and map it into that format. Also don't forget to check the `MembersTable` component to see how you'd map the `members` field. Notice that the API returns the `members` array separate from the `wallet` object, so you'd have to include it into the `wallet` object.

**Review by abdelhakeem — CHANGES_REQUESTED** [review]
Pretty good work! Just a few changes needed and we're ready to go. It'd be nice if you coordinate with @alexandredelaitre (for the API), @Apra487 (for the `MembersTable` component), and @Tess99854 (for the `WalletDetails` component) in case you have any doubts.

**Comment by abdelhakeem** [comment]
I think this is in good shape now. Just waiting for the SPL token public key to be available in the API.

**Comment by abdelhakeem** [comment]
The database and API now have an SPL token field, I think once this gets updated we can get it merged.

File: pages/manage/[pubkey].tsx (hunk 1 above)
Comment (NikhilSharma03) [reply pages/manage/[pubkey].tsx]: Resolving this.

**Comment by NikhilSharma03** [comment]
@abdelhakeem Done. Can you please review it?

**Comment by abdelhakeem** [comment]
Looks good, merging this now.

**Pull request merged**